include/lsst/afw/table/Exposure.h

```diff
@@ -1,8 +1,8 @@
 // -*- lsst-c++ -*-
-/* 
+/*
```

Comment:
I do sometimes wonder if we shouldn't just do a single clean up of all whitespace issues (tied with people making sure that trailing whitespace is visible in their editors and possibly a CI check).